docs/Sampling-batchaggregator.md

```diff
@@ -0,0 +1,70 @@
+# Sampling Aggregate
+
+
+Description
+-----------
+Sampling a large dataset flowing through this plugin to pull random records. Supports two types of sampling
+i.e, Systematic Sampling and Reservoir Sampling.
+
+
+Properties
+----------
+**sampleSize:** The number of records that needs to be sampled from the input records.
```

Comment:
please add Either of 'samplePercentage' or 'sampleSize' should be specified for this plugin.